apps/voting/app/src/components/MenuButton/MenuButton.js

```diff
@@ -1,31 +1,17 @@
 import React from 'react'
-import styled from 'styled-components'
-import { theme } from '@aragon/ui'
+import { ButtonIcon } from '@aragon/ui'
 import IconMenu from './IconMenu'
 
-const StyledButton = styled.button`
-  border: none;
-  background: none;
-  margin-right: 8px;
-  height: 32px;
-  width: 32px;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  padding: 0;
-  cursor: pointer;
-  outline: none;
-
-  &:focus {
-    border: 2px solid ${theme.accent};
-  }
-  &:active {
-    border: none;
-  }
-`
-
 export default props => (
```

Comment:
Why didn't you use this approach for all the apps?

Reply:
It is exactly the same for all the apps? [Finance](https://github.com/aragon/aragon-apps/blob/use-viewport/apps/finance/app/src/components/MenuButton/MenuButton.js) / [Token Manager](https://github.com/aragon/aragon-apps/blob/use-viewport/apps/token-manager/app/src/components/MenuButton/MenuButton.js) / [Voting](https://github.com/aragon/aragon-apps/blob/use-viewport/apps/voting/app/src/components/MenuButton/MenuButton.js)